docs/windows.md

```diff
@@ -51,18 +51,10 @@ document.
 kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/<TAG>/antrea.yml
 ```
 #### Add Windows antrea-agent and kube-proxy DaemonSets
-Now you can add Windows-compatible versions of antrea-agent and kube-proxy by
-applying yaml files:
-- antrea-windows.yml:
-  - Deploy antrea-agent Windows DaemonSet.
+Add Windows-compatible versions of kube-proxy by applying yaml file:
 - kube-proxy.yaml:
   - Deploy kube-proxy Windows DaemonSet.
```

Comment:
check the markdown rendering for this, it looks very strange and unhelpful

I don't think this line serves a purpose